COLLADAMaya/include/COLLADAMayaSyntax.h

```diff
@@ -36,7 +36,7 @@ namespace COLLADAMaya
 
     /** Profile specific extra attribute parameters. */
     static const char* PROFILE_COLLADA                              = "COLLADA";
-    static const char* PROFILE_MAYA                                 = "OpenCOLLADAMaya";
+    static const String PROFILE_MAYA                                 = "OpenCOLLADAMaya";
     static const char* PROFILE_MAX                                  = "OpenCOLLADA3dsMax";
```

Comment:
I needed PROFILE_MAYA to be a String. I'll update the others later in another PR.